Misc/NEWS.d/next/Core_and_Builtins/2025-09-15-13-14-20.gh-issue-137838.Ju-vRW.rst

```diff
@@ -0,0 +1,2 @@
+Move _PyUOpInstruction buffer[UOP_MAX_TRACE_LENGTH] to _PyThreadStateImpl.
+Patch By Donghee Na.
```

Comment:
We don't need a news I think? This is a completely internal change. Also even if we do have a news, it would be confusing.